qualtran/_infra/composite_bloq.py

```diff
@@ -771,7 +771,9 @@ def add_register(
         return None
 
     @classmethod
-    def from_signature(cls, signature: Signature, add_registers_allowed=False):
+    def from_signature(
+        cls, signature: Signature, add_registers_allowed=False
```

Comment:
nit on your nits: add_registers_allowed is missing a type hint

Reply:
done